docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -987,9 +987,10 @@ Since: 0.10.0
 RPAD(input, length, padding)
 ```
 
-Pads the input string, starting from the end, with the specified padding string until the target length is reached. If the input string is longer than the specified target length it will be truncated. 
+Pads the input string or bytes, starting from the end, with the specified padding of the same type until the target length is reached. 
+If the input is longer than the specified target length, it is truncated. 
```

Comment:
This is just GitHub formatting making spacing look inconsistent, it's consistent.